packages/myst-to-react/src/MyST.tsx

```diff
@@ -20,18 +20,24 @@ export function selectRenderer(renderers: NodeRenderersValidated, node: GenericN
   return SpecificComponent ?? componentRenderers?.base ?? DefaultComponent;
 }
 
-export function MyST({ ast }: { ast?: GenericNode | GenericNode[] }) {
+export function MyST({
+  ast,
+  className,
+}: {
+  ast?: GenericNode | GenericNode[];
+  className?: string;
+}) {
   const renderers = useNodeRenderers();
   if (!ast || ast.length === 0) return null;
   if (!Array.isArray(ast)) {
     const Component = selectRenderer(renderers, ast);
-    return <Component key={ast.key} node={ast} />;
+    return <Component key={ast.key} node={ast} className={className} />;
   }
   return (
     <>
       {ast?.map((node) => {
         const Component = selectRenderer(renderers, node);
-        return <Component key={node.key} node={node} />;
+        return <Component key={node.key} node={node} className={className} />;
```

Comment:
Interesting. You are moving this from on the node to also coming in through a prop.

Maybe ID doesn't make sense.